.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description
```

Comment:
@ManuSG8 
This is the new pull request template. Let's fill it with valuable information about the PR's changes from now on 😉